acceptance/tests/provider/package/dpkg_hold_true_package_is_latest.rb

```diff
@@ -1,4 +1,4 @@
-test_name "dpkg ensure held package is latest installed" do
+test_name "dpkg ensure hold package is latest installed" do
   confine :to, :platform => /debian-8-amd64/
```

Comment:
can you please update this to debian-9 as debian-8 was dropped in puppet-agent main